<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameEditor</class>
 <widget class="QWidget" name="FrameEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="frameLeft">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/frame_left.png</normaloff>:/images/frame_left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="frameRight">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/frame_right.png</normaloff>:/images/frame_right.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="frameTop">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/frame_top.png</normaloff>:/images/frame_top.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="frameBottom">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/frame_bottom.png</normaloff>:/images/frame_bottom.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="frameAll">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/frame_all.png</normaloff>:/images/frame_all.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="frameNone">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/frame_none.png</normaloff>:/images/frame_none.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../page_editor.qrc"/>
 </resources>
 <connections/>
</ui>
